packages/core/src/tenants/SystemContext.ts

```diff
@@ -31,6 +33,20 @@ export default class SystemContext {
           storageProviderDataGuard
         );
       })(),
+      (async () => {
+        this.experienceBlobsProviderConfig = await this.loadConfig(
+          pool,
+          StorageProviderKey.ExperienceBlobsProvider,
+          storageProviderDataGuard
+        );
+      })(),
+      (async () => {
+        this.experienceZipsProviderConfig = await this.loadConfig(
+          pool,
+          StorageProviderKey.ExperienceZipsProvider,
+          storageProviderDataGuard
+        );
+      })(),
```

Comment:
`loadProviderConfigs` looks awful now. can we just load all configs and filter them by type? maybe in a separate pull

Reply:
OK, [task](https://linear.app/silverhand/issue/LOG-9641/refactor-loadproviderconfigs-in-systemcontext) created